specification/paths/Returns-v1-reasons.json

```diff
@@ -0,0 +1,174 @@
+{
+  "get": {
+    "tags": [
+      "ReturnReasons"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "returns.manage"
```

Comment:
I don't think the `return_reasons.manage` scope is needed here. If someone wants to attach the reasons to his shop he needs to be able to view all the return reasons to be able to select the ones that he wants to attach to the shop right?

Reply:
I thought if someone only has `return_reasons.manage` and not `returns.manage` then he should be able to GET the resources they have created. I guess this will never happen and our users will have both scopes.

Anyway, it sounds logical that `return_reasons.manage` should give access to all CRUD endpoints of ReturnMethods.